tests/service/privacy_request/request_runner_service_test.py

```diff
@@ -337,6 +341,47 @@ def test_create_and_process_access_request_saas(
     pr.delete(db=db)
 
 
+@pytest.mark.saas_connector
+@mock.patch("fidesops.models.privacy_request.PrivacyRequest.trigger_policy_webhook")
+def test_create_and_process_erasure_request_saas(
+    trigger_webhook_mock,
+    connection_config_saas,
+    dataset_config_saas,
+    db,
+    cache,
+    erasure_policy_hmac,
+    generate_auth_header,
+    mailchimp_account_email,
+    reset_saas_data
+):
+    customer_email = mailchimp_account_email
+    data = {
+        "requested_at": "2021-08-30T16:09:37.359Z",
+        "policy_key": erasure_policy_hmac.key,
+        "identity": {"email": customer_email},
+    }
+
+    pr = get_privacy_request_results(db, erasure_policy_hmac, cache, data)
+
+    connector = SaaSConnector(connection_config_saas)
+    request = ("GET", "/3.0/search-members", {"query": mailchimp_account_email}, {})
```

Comment:
This request is being sent by the `reset_saas_data` fixture's setup too. Until we have an intentionally designed SaaS integrations testing solution in place, it'd be good to try and keep the number of requests sent to external services to a minimum so as to keep the test run time down, and rate limits to as low loadings as possible.

Reply:
Keeping this in mind for the future. I'd like to develop a better test suite for SaaS connectors where we mock most of the interactions with a given API and only have a single integration test with the actual API to verify that nothing has changed in the API spec.